lib/node_modules/@stdlib/array/empty-like/lib/main.js

```diff
@@ -0,0 +1,63 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2023 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var dtype = require( '@stdlib/array/dtype' );
+var empty = require( '@stdlib/array/empty' );
+var format = require( '@stdlib/string/format' );
+
+
+// MAIN //
+
+/**
+* Creates an uninitialized array having the same length and data type as a provided input array.
+*
+* @param {(Array|TypedArray|ComplexArray)} x - input array
+* @param {string} [dtype="float64"] - data type
```

Comment:
```suggestion
* @param {string} [dtype] - data type
```

There is no default.